.github/workflows/sql-test-workflow.yml

```diff
@@ -0,0 +1,103 @@
+name: SQL Plugin Tests
+
+on:
+  push:
```

Comment:
I don't think we need to run this for all branch pushes.  Maybe just for release branches?  This is a big run action with lots of artefacts and could be run manually when needed.

Reply:
I propose to run this workflow only on `dev-*` branches

Reply:
I was thinking of the opposite.  Do we want to run this on every push?

Reply:
[Hivemind](https://stackoverflow.com/a/62768214) says
> One thing that's not mentioned in the documentation: the workflow must exist on the default branch for the "Run workflow" button to appear.

I tested - GH prohibits running a workflow with `workflow_dispatch` unless it is on main. I tried GH CLI and web too.
Once it is merged we can update this workflow not to run on `push`. Or not update - depends on demand.